docs/Unused-Private-Method.md

```diff
@@ -0,0 +1,46 @@
+## Introduction
+
+Classes should use their private methods. Otherwise this is dead
+code which is confusing and bad for maintenance.
+
+## Example
+
+Given:
+
+```Ruby
+class Car
+  private
+  def self.start_the_engine; end
+  def drive; end
+end
+```
+
+`Reek` would emit the following warning:
+
+```
+2 warnings:
+  [3]:Car has the unused private class method `start_the_engine` (UnusedPrivateMethod)
```

Comment:
This warning is wrong, as `Car.start_the_engine` is not a private method (`private` does not apply to class methods, only the `private_class_method` does).